libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/SpecTranslate/Conway/Base.hs

```diff
@@ -127,31 +103,6 @@ instance SpecTranslate ctx TxIn where
 
   toSpecRep (TxIn txId txIx) = toSpecRep (txId, txIx)
 
```

Comment:
I think it makes more sense to put these into `Core`, since they're not specific to `Conway`